controllers/data.controller.js

```diff
@@ -89,7 +88,14 @@ const handlePlacementQuestions = async (
       },
     },
   );
-  return response;
+  if (response) {
+    response = {
+      tier: newTier,
+      points: 0,
+    };
+  } else {
+    return null;
```

Comment:
i think you can just return response. I think you dont need to check if (response), because if it is empty , it will just send empty array.